source/exe/BUILD

```diff
@@ -176,6 +177,45 @@ envoy_cc_win32_library(
     ],
 )
 
+envoy_cc_library(
+    name = "scm_impl_lib",
+    hdrs = select({
+        "//bazel:windows_x86_64": [
+            "win32/service_base.h",
+        ],
+        "//conditions:default": [],
+    }),
+    strip_include_prefix = select({
+        "//bazel:windows_x86_64": "win32",
+        "//conditions:default": "",
+    }),
+    deps = envoy_cc_platform_dep("scm_impl_lib"),
+)
+
+envoy_cc_posix_library(
+    name = "scm_impl_lib",
+    srcs = [],
+    hdrs = [],
+    deps = [],
+)
```

Comment:
I'm no bazel expert but can't everything just be rolled into this rule with selects?